packages/react-liveness/src/components/FaceLivenessDetector/LivenessCheck/LivenessCameraModule.tsx

```diff
@@ -116,7 +116,7 @@ export const LivenessCameraModule = (
   const isFaceMovementChallenge =
     useLivenessSelector(selectChallengeType) === FACE_MOVEMENT_CHALLENGE.type;
   const allowSelectableDevicesMobile =
```

Comment:
This is controlling whether we allow selectableDevices regardless of whether it's mobile or not, right? So should this just be `allowSelectableDevices`?

Reply:
Update the names to be a little more clear and added a comment